terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1936,20 +1936,20 @@ def merge(sets: Iterable[Iterable[str]]) -> Iterable[str]:
                                     )
                                     for command in [
                                         sq(
+                                            # docker volume prune --force --all
```

Comment:
This comment doesn't seem useful.

Reply:
Ah, that was notes-to-self unintentionally left in.